variables.tf

```diff
@@ -32,6 +32,12 @@ variable "lambda_package" {
   default     = "anti-virus"
 }
 
+variable "memory_size" {
+  description = "Lambda memory allocation, in MB"
+  type        = string
+  default     = 1536
```

Comment:
This is all based on the https://github.com/upsidetravel/bucket-antivirus-function code which I also help maintain now. They suggest just going higher to 2048.

Reply:
Where do they say that? The README still says 1024.

Reply:
I may have written that README. There was some issue or PR which had some conversation about bumping the memory limit in the last couple months. I am having trouble finding it, but I remember folks complaining about memory on this module before. And as ClamAV changes so do the requirements for this lambda.